fs-core/main/default/lwc/formulaShareRulesListView/formulaShareRulesListView.js

```diff
@@ -7,6 +7,7 @@ import getTreeGridData from '@salesforce/apex/FormulaShareRulesListViewControlle
 import recalculateSharing from '@salesforce/apex/FormulaShareRulesListViewController.recalculateSharing';
 import activateDeactivate from '@salesforce/apex/FormulaShareRulesListViewController.activateDeactivate';
 import getNamespacePrefix from '@salesforce/apex/FormulaShareUtilities.getNamespacePrefix';
+import isContactSharingControlledByAccount from '@salesforce/apex/FormulaShareUtilities.isContactSharingControlledByAccount';
```

Comment:
I think this line can be removed as it's not referenced anywhere else

Reply:
Removed.